byterun/caml/page_table.h

```diff
@@ -0,0 +1,42 @@
+#ifndef CAML_PAGETABLE_H
```

Comment:
You need to add a copyright header. I suggest copying the one from memory.h since there is no original code in this new file.